int32 id
std_msgs/Header header
geometry_msgs/Pose pose
geometry_msgs/Twist twist_local
geometry_msgs/Twist twist_body